src/panels/lovelace/editor/view-editor/hui-edit-view.ts

```diff
@@ -215,9 +215,9 @@ export class HuiEditView extends LitElement {
     this._saving = true;
 
     const viewConf: LovelaceViewConfig = {
-      ...this._config,
-      badges: this._badges!.map((entityConf) => entityConf.entity),
       cards: this._cards,
+      badges: this._badges,
+      ...this._config,
```

Comment:
Please restore the order

Reply:
You just wanted to use the new multi-line comments, didn't you? 😄